### ISSUE TITLE
CVMFS installation on the host required?

### ISSUE THREAD

**Issue by vpenso** [cvmfs-contrib/cvmfsexec#15]
Starting from the latest CentOS 7:

```shell
# host OS
>>> grep -i pretty /etc/os-release ; uname -r
PRETTY_NAME="CentOS Linux 7 (Core)"
3.10.0-1127.el7.x86_64
# install Singularity from EPEL
>>> sudo yum install -y epel-release git
>>> sudo yum install -y singularity
>>> singularity --version
singularity version 3.5.3-1.1.el7
>>> grep uid /etc/singularity/singularity.conf
# Should we allow users to utilize the setuid program flow within Singularity?
allow setuid = yes
```

Following the example in the README:

```bash
tmp=$(mktemp -d /tmp/cvmfsexec_test.XXXXX)
git clone https://github.com/cvmfs-contrib/cvmfsexec $tmp && cd $tmp
./makedist -s default
export SINGCVMFS_REPOSITORIES=alice.cern.ch
export SINGCVMFS_IMAGE="docker://centos:7"
./singcvmfs ls /cvmfs/alice.cern.ch
```

Will result in following error:

```
WARNING: skipping mount of /tmp/cvmfsexec_test.VWQPK/dist/etc/auto.cvmfs: no such file or directory
FATAL:   container creation failed: mount /tmp/cvmfsexec_test.VWQPK/dist/etc/auto.cvmfs->/etc/auto.cvmfs error: while mounting /tmp/cvmfsexec_test.VWQPK/dist/etc/auto.cvmfs: mount source /tmp/cvmfsexec_test.VWQPK/dist/etc/auto.cvmfs doesn't exist
```

This implies that the hosting node needs CVMFS to be installed:

```bash
# install CVMFS from Cern
sudo yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
yum install -y cvmfs
```

Which indeed resolve this issue.

```bash
>>> ./singcvmfs ls /cvmfs/alice.cern.ch
bin	     containers  el5-x86_64  el7-x86_64  fah   ubuntu1404-x86_64  x86_64-2.6-gnu-4.1.2	x86_64-2.6-gnu-4.8.3
calibration  data	 el6-x86_64  etc	 java  ubuntu1604-x86_64  x86_64-2.6-gnu-4.7.2	x86_64-2.6-gnu-4.8.4
```

Is this dependency intentional?

**Comment by DrDaveD** [comment]
Thank you for the report.  It was not intentional.  It's true that my testing machine had cvmfs installed however, and I can reproduce the problem on a machine that does not.  I see the problem is that dist/etc/auto.cvmfs is a symlink to a full path.  I will work on a fix today.

**Comment by DrDaveD** [comment]
Fixed in #16 and release 3.2.  Please confirm.

**Comment by vpenso** [comment]
Works...

Thanks a lot
